snooty.toml

```diff
@@ -29,7 +29,7 @@ aagent = "Automation Agent"
 adf = "Atlas Data Federation"
 atlas-cli = "Atlas CLI"
 atlas-cli-full = "MongoDB Atlas CLI"
-atlas-cli-version = "1.19.0"
+atlas-cli-version = "1.20.0"
```

Comment:
Please update the redirects file as well (Add a new version, Update the symlinks for the master and stable branches)